include/bluetooth/direction.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_DF_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_DF_H_
+
+/** @brief Constant Tone Extension parameters for connectionless
+ * transmission.
+ *
+ * The structure holds information required to setup CTE transmission
+ * in periodic advertising.
+ */
+struct bt_le_df_adv_cte_tx_params {
```

Comment:
I think we should not add `_le_` as infix here, in order to get shorter API names since this is completely new.

Also the other places in bluetooth.h we use `param` instead of `params` so should align there (I noticed gatt.h uses it different, which is a shame, but this should be aligned with bluetooth.h IMO).

Reply:
infix removed here and in `bt_df_set_adv_cte_tx_params` function name.